webview/src/experiments/components/table/styles.module.scss

```diff
@@ -203,13 +203,14 @@ $badge-size: 0.85rem;
 // table head styles
 
 .tableIndicators {
```

Comment:
@sroy3, @julieg18 I've moved the `<Indicators />` from the global scope into `<TableHeaderCell />` are we ok with this?